src/lib/util.ts

```diff
@@ -50,7 +50,8 @@ export async function infoLogger(info: unknown): Promise<void> {
 
 export function toFixed(num: number, precision: number): number {
   if (!num) return 0
-  return parseFloat(num.toFixed(precision))
+  // don't remove `Number` cast, it is required. Thanks TypeScript!
```

Comment:
Why is number cast necessary if input is supposed to be number already? Looks like a bug elsewhere to me.

Reply:
Exactly. when I print the type of `num`, it was string??? 
Isn't typescript forcing it to be `number`?...

Reply:
There are a few strictness checks in TypeScript, I think not all of them might be enabled or there is `as number` or something like that somewhere that compiler will blindly believe even if it is not true (just like with `unsafe` in Rust).

So better find where this function is called from and fix that instead of hacking here.